modules/nf-core/metamaps/classify/main.nf

```diff
@@ -0,0 +1,68 @@
+
+process METAMAPS_CLASSIFY {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/metamaps:0.1.633d2e0--h21ec9f0_0':
+        'biocontainers/metamaps:0.1.633d2e0--h21ec9f0_0' }"
+
+    input:
+    tuple val(meta), path(classification_res)
+    tuple val(meta), path(meta_file)
+    tuple val(meta), path(meta_unmappedreadsLengths)
+    tuple val(meta), path(para_file)
```

Comment:
Where are these inputs used?

Reply:
Here is an example of how the module looks like. 

```
metamaps mapDirectly -t 5 --all -r databases/miniSeq+H/DB.fa -q test.fastq.gz  -o classification_results --maxmemory 60
metamaps classify -t 5 --mappings classification_results --DB databases/miniSeq+H
```

So all those files are needed and can be found under the folder `classification_results` which is the output of the module `metamaps/mapDirectly`

Is it more clear?

Reply:
Yes, thank you! I think in this case it would make it more clear to add all of these into the same input channel since they all come from the same process. Can you also add these values in the meta.yaml?

Reply:
I tried to do that but the nf-test was failing and decided to go ahead with a bit more ugly way :) 

So you mean in `metamaps/classify`:

```
input:
tuple val(meta), path(classification_res), path(meta_file), path(meta_unmappedreadsLengths), path(para_file)
path database_folder
```

How does the nf-test should look like? This is where I find it confusing.

Reply:
You can join the output channels from the first process. (https://www.nextflow.io/docs/latest/operator.html#join).

Reply:
Let me know if you need an example :)

Reply:
Thank you! Updated the PR now.